libs/runtime/ebpf_epoch.c

```diff
@@ -1028,3 +1059,79 @@ _ebpf_epoch_work_item_callback(_In_ cxplat_preemptible_work_item_t* preemptible_
 
     cxplat_release_rundown_protection(&_ebpf_epoch_work_item_rundown_ref);
 }
+
+/**
+ * @brief Add the CPU to the next active CPU table.
+ *
+ * @param[in] cpu_id CPU to add.
+ */
+static void
```

Comment:
Need to get current epoch when CPU goes active to avoid tagging freed memory with wrong epoch.